roles/vault/tasks/main.yml

```diff
@@ -45,3 +45,12 @@
     vault
 
 - include: bootstrap.yml
+
+- name: set vault_token fact
+  set_fact:
+    vault_token: "{{ vault_root_token }}"
+  when: >
+    "{{ vault_root_token is defined
+    and 'Vault is already initialized' not in vault_init_output.stdout }}"
```

Comment:
I think this needs to be:
```
and 'Vault is already initialized' in vault_init_output.stdout }}"
```
The `extract root token` task handles the `not in` case.